tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
```

Comment:
As part of this PR, let's add a `feature` changelog entry to reflect that we'll be bumping the embedded Python interpreter to 3.11